.atlascli.goreleaser.yml

```diff
@@ -14,7 +14,7 @@ builds:
       binary: bin/atlas
       main: ./cmd/atlas/atlas.go
       ldflags:
-        - -s -w -X github.com/mongodb/mongocli/internal/version.Version={{.Version}} -X github.com/mongodb/mongocli/internal/version.GitCommit={{.FullCommit}}
+        - -s -w -X github.com/mongodb/mongocli/internal/version.Version={{ .Env.VERSION_GIT }} -X github.com/mongodb/mongocli/internal/version.GitCommit={{.FullCommit}}
```

Comment:
why is this needed? go releaser should be handling this correctly if they don't let's open an issue for them

Reply:
yes, you are right. Thanks!